test/index.ts

```diff
@@ -165,3 +212,26 @@ function createWait(name: string, delay: number): () => Promise<void> {
         });
     };
 }
+
+/**
+ * @internal
+ *
+ * Extracts the seed and path from a fast-check error message and formats them into a command
+ * that can be used to reproduce the exact test case that failed.
+ *
+ * @param error - The error message from fast-check containing seed and path information
+ * @returns A command string that can be used to reproduce the exact test case that failed
+ */
+function formatPropertyTestReproductionCommand(error: string): string {
+    // Look for the fast-check error details in the full stack trace
+    const fcErrorMatch = error.match(
+        /Property failed after \d+ tests[\s\S]*?{ seed: (-?\d+), path: "([^"]+)"/
+    );
+
+    if (fcErrorMatch === null) {
+        return 'Could not parse fast-check error output';
+    }
+
+    const [, seed, path] = fcErrorMatch;
+    return `AZLE_PROPTEST_SEED=${seed} AZLE_PROPTEST_PATH="${path}" npm test`;
```

Comment:
Would we want to add `AZLE_VERBOSE=true` and `AZLE_TEMPLATE=true` in here as well?